go.mod

```diff
@@ -230,6 +230,7 @@ require (
 	go.starlark.net v0.0.0-20200306205701-8dd3e2ee1dd5 // indirect
 	go.uber.org/atomic v1.10.0 // indirect
 	go.uber.org/multierr v1.11.0 // indirect
+	golang.org/x/exp v0.0.0-20220722155223-a9213eeb770e // indirect
```

Comment:
Hrmm, what pulled this in? The description of this module is not encouraging for a dependency:

> Warning: Packages here are experimental and unreliable

Reply:
I actually thought we already added this one to the unwanted dependencies json file to keep from accidentally picking it up

Reply:
Looks like it came in with the ANTLR runtime roughly 9+ months ago. I'm not sure why it's just showing up now though. The function being used is [`slices.EqualsFunc()`](https://pkg.go.dev/golang.org/x/exp/slices#EqualFunc).

It looks pretty easy to replace, and I can submit a fix upstream to ANTLR, but I'm not sure when it will take effect.

Reply:
https://github.com/antlr/antlr4/pull/4294

Reply:
I tried adding `"golang.org/x/exp": []` to `hack/unwanted-dependencies.json` but got `Good news! Unwanted dependency "golang.org/x/exp" is no longer referenced. Remove status.unwantedReferences["golang.org/x/exp"] in unwanted-dependencies.json to ensure it doesn't get reintroduced.`

Reply:
Figured it out. I've added `golang.org/x/exp` to `hack/unwanted-dependencies.json`.